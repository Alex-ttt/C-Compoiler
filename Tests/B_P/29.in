int func(int ***a){
	return 0;
}

void main(){
	int ***q;
	func(q);
}