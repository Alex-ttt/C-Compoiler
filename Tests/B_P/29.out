 Symbols Table:
   func  int function func(
          a pointer to pointer to pointer to int
  )
    {
      Statements:
      return
        0
    }
   main  void function main(
  )
    {
      Statements:
      func
      (
        q
      )
      Symbols Table:
              q pointer to pointer to pointer to int
    }
